test/Microsoft.OpenAPI.OData.Reader.Tests/Resources/TripService.OpenApi.V2.json

```diff
@@ -1167,14 +1167,12 @@
           {
             "in": "path",
             "name": "lat",
-            "required": true,
-            "format": "double"
+            "required": true
```

Comment:
the format of this one is still missing. Maybe something was missed in the OpenAPI.Net lib?
Other than that, the rest of v2 and v3 looks good.

Reply:
That's because we in Parameter serialization for v2, we aren't serializing the schema inside this condition: https://github.com/microsoft/OpenAPI.NET/blob/eb4778d1caf753eab32ce71517b3a02b89a7d46f/src/Microsoft.OpenApi/Models/OpenApiParameter.cs#L333-L335

This means that we are also not getting the oneOf/allOff composition into oneOf.

Reply:
And I believe this should be addressed since the format property is supported on parameters.

Reply:
You were right, we had missed something in the fix that was done in the OpenAPI.NET. Here's the PR to remedy that: https://github.com/microsoft/OpenAPI.NET/pull/1001

Reply:
thanks for making the fix. resolving